### ISSUE TITLE
Multiple Quest changes

### ISSUE THREAD

**Issue by xtake** [GTNewHorizons/GT-New-Horizons-Modpack#11877]
### Your GTNH Discord Username

Glueky#6757

### Your Pack Version

2.2.0.0

### Your Proposal

_Those are all my opinions feel free to comment on them._

### Quests:

- [ ]  1. Time to Kill (Wither)/Diamond Dolly
Diamond Dolly Quest (Lv/# 626) is dependent on Time to Kill (Wither). Which is dependent on killing all vanilla mobs quests. Shouldn't be dependent on all of them. For me I'm missing all 10 Witch kills and Enderman kills (Enderpearls I just got by smelting enderman). 
Luckily Nasa workbench which also needs nether star plates dsn't need those quests but still should be fixed somehow.
_Suggestion_: Remove dependencies for either Wither quest or Dolly quest

- [ ]  2. Night vision quest (Novice Thaumcraft)
Mostly optional already. Last is retrieval of night vision 3 minutes. I made night vision 8 min because of backpack so it didn't count. 
_Suggestion_: make it retrieval any kind of night vision potion. 

- [ ]  3. Pure Bees / Bee Breeding
Bee Breeding questline always needs pure bees (Same species active/inactive). Sometimes early bees (Industries/Imperial lines) you breed very fast and only skim over some bee species.
Obviously you will need the pure ones later anyway but I for example was breeding bees and only got common/forest, common/meadows and then cultivated/common or something like that.
_Suggestion_: If possible make it only need active trait. (No idea if it's possible. Not a big deal if it isn't)

- [ ]  4. Nasa Workbench dependencies 
Nasa Workbench needs IV circuits which is a quest in HV chapter. I forgot it needed IV circuits so was looking around for Space Race Chapter but couldn't find it. 
_Suggestion_: If possible make the Nasa Workbench Quest/Space Race Chapter appear before you complete IV circuit quest so you know what you are working towards. Maybe same quest shown in both chapters. 

- [ ]  5. Oil Reservoirs (Working with Oil # 2996)
It needs Seismic Prospector and 30 oil cells. I personally don't like the seismic prospector at all. I'd just make a prospector scanner usually.
_Suggestion_: Make it either block version or handheld version as completion for quest.

- [ ]  6. Traveller's Gear (# 1781)
Out of the 6 traveller's gears I'd usually only make the gloves for faster mining. Maybe the Goggles for night vision.
But having to craft/have all 6 gears just to get to the upgrading gear quests is kinda stupid.
_Suggestion_: Either fully remove the quest # 1781 or make it a checkbox task with some information about them all. And then split the upgrade quest into first getting it then upgrading or something similar. (Maybe also remove the dependency on diamond armor then. Gloves don't need it for example)

- [ ]  7. Night vision goggles (# 776) / Raw Carbon Mesh/MV (# 773)
The goggles are locked behind Carbon parts which is already stupid because I just want it for my backpack. 
But then the Carbon Parts/ Raw Carbon Mesh Quests are locked behind LV autoclave. Me being in HV and having a HV autoclave quests obviously wouldn't go back and make a lv autoclave normally. 
_Suggestion_: Remove LV Autoclave dependency and/or remove carbon parts dependency for night vision goggles. 

- [ ]  8. Unknown Seeds No More (# 721)
I personally don't like the scanner from GT and only use it for assline or other things.
For scanning crops the cropalyzer. For bees the beealyzer and for the rest the portable scanner.
But crop and portable scanner (as well as other things) are locked behind the gt scanner quest.
Also Weeds II quest needs GT Scanner quest why?
_Suggestion_: Make the GT scanner a optional thing in the Unknown seeds quests so people can continue that questline without having to make a machine they don't really want. (Also make the Field Kit # 503 optional. No idea who uses that at all. Beealyzer is cheap and cool)

- [ ]  9. Hearty Breakfast (# 1089)
Quest is dependent on Juice Quest. Which needs 7 different kinds of juice. It's not hard to do but usually if you make hearty breakfast for living axe you'll just make 1 kind of juice and be done with it.
_Suggestion_: Make the Juice Quest only need 1 juice or make hearty breakfast not dependent on it.

### Earlier Tier completions:

- [ ]  MV Bending Machine
You can easily use LV Bending machine till HV. I only made HV bending for enchanting table currently. 
But some optional quests are locked behind MV Bending. 
_Suggestion_: Remove dependencies or auto complete MV with HV bending. 

- [ ]  LV Cutting Machine
I didn't need a cutting machine till MV Silicon Wafer's. Some optional quests locked behind LV one tho.
_Suggestion_: Same as above.

- [ ] LV Autoclave
Already mentioned above in  _7. Night Vision Goggles._
_Suggestion_: Auto-complete with HV or remove dependencies. 

- [ ]  MV Extruder
MV Extruder wants all the shapes. I personally wouldn't make some of them. And some optional quests are locked behind this.
_Suggestion_: Make the shapes optional or let HV extruder auto complete the quest (99.995% of people will make the mv extruder in mv so it wouldn't really be a problem of free rewards)

### Earlier Tier Completion different:

No idea if it's possible with BQ3.
For people that want questbook completion. Have an optional quest or something that completes earlier tier quests but removes rewards so you have it completed but don't get free stuff.
So what I'm saying is if you make HV macerator you can choose to let it complete LV and MV macerator quests but you don't get rewards for LV and MV. 
Or you choose not to then you can do it later and get the rewards. (Which is probably worth nothing at that point but it's still free stuff)

### Luv/Zpm+ Quests:

A lot of things in ZPM+ and some in LuV don't have any quest rewards. When the ZPM+ loot bags only had diamonds in them that was fine but why not add a few more loot bags in those quests or some other small things.
I always enjoy getting some free stuff even if it's just small stuff like a couple ingots I just crafted a ton of, a few circuit parts I just crafted or something idk.

Just saw that there are barely rewards in "Powerful Nuclear Physics". Maybe add something there too. 

### Your Goal

Better Quests.

### Your Vision

Bessere Quests.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this change please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members think it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by chochem** [comment]
1. Time to Kill (Wither)/Diamond Dolly: yea, I never liked that either. Wither quest should not be gated by anything other than wither skellies.

2. Night vision quest (Novice Thaumcraft): The quest literally talks about that? As you make the extended from the normal, this all looks good as is. 

3. Pure Bees / Bee Breeding: This is on purpose and I for one think its better the way its is. You havent really bred the bee unless its pure. It would also be a crazy amount of work to change.

4. Nasa Workbench dependencies: That should be fixed if you update to 2.2.3 as visibility stuff was kinda reverted. So you can see the tabs again.

Earlier Tier Completion different:
>  Have an optional quest or something that completes earlier tier quests but removes rewards

that is not possible. Cant have quest completion without rewards.

Luv/Zpm+ Quests:
> A lot of things in ZPM+ and some in LuV don't have any quest rewards.

That was on purpose initially. If you reach LuV+ you are good enough to be able to do stuff fully yourself. However some rewards and lootbags have since been added by other devs, so I guess its inconsistent now. I don't think we really have a consensus there at the moment.


will look at more later. :)

**Comment by xtake** [comment]
1. You didn't tick the Wither quest but you agree with me. Do you want to hear opinions from others? Otherwise I would just make it dependent on just Wither Skeleton or maybe not even that. Some people just create cursed earth mob farm early because wither skellies sometimes have horrible spawn rate. But either way I think is fine.

2. About Night vision potions I get what you are saying. I just meant I straight up went for long lasting one for backpack and then realized the quest wanted the short version. It's fine tbh 

3. I get where you coming from from bees and essentially you will get the pure ones eventually because you always want to breed to pure. Was just a random thought

6. Mhh yeah I understand. Should ask a few more people what they think then either split it up or leave it.

To MV Bending, LV Cutting I think you misunderstood me. I'm just saying I wouldn't make them in that age because it seems useless. But the optional quests are locked behind the machines in those ages. 
It dsn't really matter tho because it's only optional quests. 

MV Extruder I never said I didn't make the MV Extruder. I think I did that first thing before even electrolyzer this playthrough. 
But with the shapes if you want to leave that in I understand, I'd change a few shapes, essentially what you already said and then be done with it.

**Comment by chochem** [comment]
for wither quest: I didnt tick it because I havent done it yet.
